app/tests/test_strutil.c

```diff
@@ -126,6 +127,19 @@ static void test_xstrjoin_truncated_after_sep(void) {
     assert(!strcmp("abc de ", s));
 }
 
+static void test_strquote(void) {
+    const char *s = "abcde";
+    char *out = strquote(s);
+
+    // add '"' in the begin and the end, also check out is null terminated
+    assert((strlen(s) + 2) == strlen(out));
+
+    // check only add '"' in the begin and the end
+    assert(!strncmp(s, out + 1, strlen(s)));
```

Comment:
I suggest to replace this by a single `strcmp()` call:

```diff
diff --git a/app/tests/test_strutil.c b/app/tests/test_strutil.c
index f4afabd..1489b1b 100644
--- a/app/tests/test_strutil.c
+++ b/app/tests/test_strutil.c
@@ -131,11 +131,8 @@ static void test_strquote(void) {
     const char *s = "abcde";
     char *out = strquote(s);
 
-    // add '"' in the begin and the end, also check out is null terminated
-    assert((strlen(s) + 2) == strlen(out));
-
-    // check only add '"' in the begin and the end
-    assert(!strncmp(s, out + 1, strlen(s)));
+    // add '"' at the beginning and the end
+    assert(!strcmp("\"abcde\"", out));
 
     SDL_free(out);
 }
```

It catches more possible errors (it asserts the expected output, not only some of its characteristics).